kuma/wiki/management/commands/clean_document.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+Manually schedule the cleaning of one or more documents
+"""
+from __future__ import division
+from __future__ import unicode_literals
+
+from math import ceil
+
+from celery import chain
+from django.core.management.base import BaseCommand, CommandError
+
+from kuma.core.utils import chunked
+from kuma.users.models import User
+from kuma.wiki.models import Document
+from kuma.wiki.tasks import (clean_document_chunk, email_document_progress)
+
+
+class Command(BaseCommand):
+    args = '<document_path document_path ...>'
+    help = 'Clean the current revision of one or more wiki documents'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'paths',
+            help='Path to document(s), like /en-US/docs/Web',
+            nargs='*',  # overridden by --all or --locale
+            metavar='path')
+        parser.add_argument(
+            '--all',
+            help='Clean ALL documents (rather than by path)',
+            action='store_true')
+        parser.add_argument(
+            '--locale',
+            help='Clean ALL documents in this locale (rather than by path)')
+
+    def handle(self, *args, **options):
+        user = User.objects.get(username='mdnwebdocs-bot')
```

Comment:
Nit: This user could be created if it doesn't exist, which would make it easier to ensure it doesn't have a password, is in known authors, etc.